packages/hub/plugin-loader.js

```diff
@@ -21,6 +21,7 @@ const featureTypes = [
   'computed-field-types',
   'writers',
   'searchers',
+  'schemas',
```

Comment:
People don't always agree on what the plural of "schema" should be, but this is fine.